web/app/js/components/ResourceDetail.jsx

```diff
@@ -174,6 +168,13 @@ export class ResourceDetailBase extends React.Component {
         )
       ];
 
+    if (resource.type !== "pod") {
```

Comment:
It would be helpful to add a comment above this line explaining why you are differentiating resource types.